shaders/glsl/renderchunk.fragment

```diff
@@ -409,7 +388,7 @@ void main()
 	const vec3 moonLightColor = vec3(0.5, 0.75,1.0) * 0.25;
 	
 	
-	//blinn-phong
+	// Blinn-phong (?)
```

Comment:
Blinn-phong is a model of specular lighting

Reply:
Ok, I'll remove the mark from those two